<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StuWindow</class>
 <widget class="QMainWindow" name="StuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>1000</width>
      <height>650</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>650</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>650</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;仿宋&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>请假</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>40</y>
        <width>263</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_id">
         <property name="minimumSize">
          <size>
           <width>54</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>学号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_id">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>263</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_class">
         <property name="minimumSize">
          <size>
           <width>54</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>班级</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_class"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_reason">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>230</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>请假理由：</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_reson">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>270</y>
        <width>591</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_apply">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>530</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>申请</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>263</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_name">
         <property name="minimumSize">
          <size>
           <width>54</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>姓名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_name">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>281</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_xueyuan">
         <property name="text">
          <string>学院</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_xueyuan">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>计算机科学与工程学院</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>光电学院</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>理学院</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>外国语学院</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>材料与化学学院</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>机械工程与自动化学院</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>450</y>
        <width>591</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>请假时间</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit">
         <property name="displayFormat">
          <string>yyyy-MM-dd hh:mm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>至</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit_2">
         <property name="displayFormat">
          <string>yyyy-MM-dd hh:mm</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>530</y>
        <width>251</width>
        <height>29</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="pushButton_quit">
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_reset">
         <property name="text">
          <string>重置</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>180</y>
        <width>211</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string> 审批老师</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_tea"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>请假记录</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_operator">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>10</y>
        <width>120</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>操作</string>
      </property>
      <widget class="QPushButton" name="pushButton_modify">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>修改信息</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_delete">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>删除信息</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_back">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>撤销申请</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>831</width>
        <height>591</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>请假流水ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>请假原因</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>请假时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>申请状态</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_quit2">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>570</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
